CRM/EventsExtras/Hook/BuildForm/EventTabHeader.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+use CRM_EventsExtras_SettingsManager as SettingsManager;
+
+/**
+ * Class for Event TabHeader
+ */
+class CRM_EventsExtras_Hook_BuildForm_EventTabHeader {
+
+  /**
+   * Hides Tabs based on settings
+   *
+   * @param string $formName
+   * @param object $form
+   */
+  public function handle($formName, &$form) {
+    if (!$this->shouldHandle($formName, $form)) {
+      return;
+    }
+    $this->hideTab($form);
+  }
+
+   /**
+   * Checks if the hook should be handled.
+   *
+   * @param string $formName
+   * @param object $form
+   *
+   * @return bool
+   */
+  protected function shouldHandle($formName, &$form) {
+    if ($form instanceof CRM_Event_Form_ManageEvent) {
+
+      return TRUE;
+    }
+    return FALSE;
+  }
+
+  private function hideTab(&$form){
+    $vars = $form->get_template_vars('tabHeader');
+    $locationTabSetting =  SettingsManager::SETTING_FIELDS['LOCATION_TAB'];
```

Comment:
Seems there are extra spaces before and after the `=` operator. Please check below too.